client/src/components/Navbar/Navbar.tsx

```diff
@@ -112,10 +113,13 @@ const Navbar: React.FC = () => {
   };
 
   const renderMenuItems = () => {
-    // TODO: conditionally render based on profile type
     return menuItems.map((menu) => {
       if (menu.authenticated) {
-        return loggedInUser && <MenuItem key={menu.resource} {...menu} />;
+        if (profile) {
+          if (!menu.canView || menu.canView == profile.accountType || menu.canView.length > 1) {
```

Comment:
Not quite the approach that was expected. If the account type is included in the canView list then render the route.